k8s_helm_charts/zdm/templates/secrets.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
```

Comment:
I would not advise the secret lifecycle being tied to the helm release lifecycle. Better to get it out of the helm chart, or at least make it optional here

Reply:
good point, also saw your earlier comment in the README.md. I'll adjust them.

Reply:
Should be addressed in the latest commit.